tests/Fixtures/TestBundle/Entity/CompositeRelation.php

```diff
@@ -50,7 +50,7 @@ class CompositeRelation
      */
     public function getId()
     {
-        return sprintf('%s-%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
+        return sprintf('%s;%s', $this->compositeItem->getId(), $this->compositeLabel->getId());
```

Comment:
A separate `id` property? Where does it come from when there's no `getId` method?